File: c#/src/Docmost.Sdk/Models/UpdateUserDto.cs
namespace Docmost.Sdk.Models;

public class UpdateUserDto
{
    [System.Text.Json.Serialization.JsonPropertyName("name")]
    public string? Name { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("email")]
    public string? Email { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("avatarUrl")]
    public string? AvatarUrl { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("fullPageWidth")]
    public bool? FullPageWidth { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("pageEditMode")]
    public string PageEditMode { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("locale")]
    public string? Locale { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("confirmPassword")]
    public string? ConfirmPassword { get; set; }

}

File: go/docmost/models/update_user_dto.go
package models

// UpdateUserDto from OpenAPI.
type UpdateUserDto struct {
	Name *string `json:"name,omitempty"`
	Email *string `json:"email,omitempty"`
	AvatarUrl *string `json:"avatarUrl,omitempty"`
	FullPageWidth *bool `json:"fullPageWidth,omitempty"`
	PageEditMode *string `json:"pageEditMode,omitempty"`
	Locale *string `json:"locale,omitempty"`
	ConfirmPassword *string `json:"confirmPassword,omitempty"`
}
